Source/Android/jni/SkylanderConfig.cpp

```diff
@@ -124,7 +126,10 @@ Java_org_dolphinemu_dolphinemu_features_skylanders_SkylanderConfig_createSkyland
   std::string file_name = GetJString(env, fileName);
 
   auto& system = Core::System::GetInstance();
-  system.GetSkylanderPortal().CreateSkylander(file_name, sky_id, sky_var);
+  std::unique_ptr<IOS::HLE::USB::SkylanderFigure> figure =
+      std::make_unique<IOS::HLE::USB::SkylanderFigure>(IOS::HLE::USB::SkylanderFigure(file_name));
```

Comment:
`make_unique` makes the type obvious so this can just be `auto` and still be readable.